src/main/java/jenkins/plugins/http_request/auth/FormAuthentication.java

```diff
@@ -71,5 +71,10 @@ public FormValidation doCheckKeyName(@QueryParameter String value) {
         public String getDisplayName() {
             return "Form Authentication";
         }
+
+        public String getHttpRequestHelpPath() {
+            return "/descriptor/"+HttpRequest.class.getName()+"/help";
```

Comment:
Sounds like you got this working. FYI, more conventional is to keep shared resources in `src/main/webapp/help/`, allowing them to be accessed using simple URLs like `/help/whateverSubdir/something.html`.